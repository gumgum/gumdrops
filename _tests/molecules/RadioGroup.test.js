import renderer from 'react-test-renderer';
import React from 'react';
import RadioGroup from '../../components/molecules/RadioGroup';

const defaultProps = {
    options: [
        {
            label: 'Cool',
            value: 'cool'
        }
    ],
    className: 'custom-class',
    context: 'success',
    name: 'radio-buttons',
    onBlur: jest.fn(),
    onChange: jest.fn(),
    size: 'sm'
};

test('Expect <RadioGroup> to render properly', () => {
    const tree = renderer.create(<RadioGroup {...defaultProps} />).toJSON();
    expect(tree).toMatchSnapshot();
});
