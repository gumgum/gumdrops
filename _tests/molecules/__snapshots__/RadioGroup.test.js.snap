// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect <RadioGroup> to render properly 1`] = `
<div
  className="gds-form-group__radio custom-class gds-form-group__radio--sm"
>
  <label
    className="gds-form-group gds-form-group__radio-label -m-r-4 gds-form-group--success"
  >
    <input
      className="gds-form-group__radio-input"
      defaultChecked={false}
      name="radio-buttons"
      onBlur={[MockFunction]}
      onChange={[MockFunction]}
      type="radio"
      value="cool"
    />
    <span
      className="gds-form-group__radio-indicator"
    />
    Cool
  </label>
</div>
`;
